lib/cylc/profiling/profile.py

```diff
@@ -63,7 +63,7 @@ def __str__(self):
 
 class ProfilingKilledException(SuiteFailedException):
     """Exception to handle the event that a user has canceled profiling whilst
-    a suite is runnnig."""
```

Comment:
Reminds me, Bob Dylan originally went by the name "Elston Gunnn" - with 3 n's ... brilliant!

Reply:
TIL!